reports/Maryland.MD

```diff
@@ -0,0 +1,9 @@
+## Prince George County | (Believed to be) June 2nd.
```

Comment:
Shouldn't the date be after the incident title?

Reply:
Also, I am curious as to what happens when `(Believed to be) June 2nd.` data gets merged, how does the API handle this input? Is it able to properly parse data and generate `2020-06-02`?